src/rust-2021/warnings-promoted-to-error.md

```diff
@@ -46,4 +46,5 @@ or `ellipsis_inclusive_range_patterns` and you've not allowed these lints throug
 use of `#![allow()]` or some other mechanism, then there's no need to migrate.
 
 To automatically migrate any crate that uses `...` in patterns or does not use `dyn` with
-trait objects, you can run `cargo fix --edition`.
\ No newline at end of file
+trait objects, you can run `cargo fix --edition` you can manually change `...` to `..=` and 
+add `dyn` before trait objects.
```

Comment:
This seems like a run-on sentence.  Was there something that was supposed to separate the statement about running `cargo fix` and the instructions for manually updating?

Reply:
Sorry - That change somehow snuck in after I thought I had reverted it!